### PR TITLE
Fix 244

### LINKED ISSUE #244: router generator should not require trailing slash on Mount
I think Router Generator should not require trailing slashes on `@Mount` annotation anymore
after [fix: mount remove trailing slash requirement #146](https://github.com/google/dart-neats/pull/146) 

I think the area to change is [this](https://github.com/dart-lang/shelf/blob/05f42601d22c9bfe490ceda50e812f83b7d1de77/pkgs/shelf_router_generator/lib/src/shelf_router_generator.dart#L283
)

### PR DESCRIPTION
**Pull request by jonasfj**
Fix #244

### CODE REVIEW

File: pkgs/shelf_router_generator/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## v1.1.0
+* Remove trailing slash requirement when using `mount`.
```
Comment (natebosch) [inline pkgs/shelf_router_generator/CHANGELOG.md:2]: [nit]
```suggestion
## v1.1.0

* Remove trailing slash requirement when using `mount`.
```

**Review by natebosch — APPROVED** [review]
Can you expand the commit message when this gets merged?

**Comment by natebosch** [comment]
Should this get merged?

**Comment by kevmoo** [comment]
@jonasfj ?

**Comment by jonasfj** [comment]
Yeah, should try to remember it tomorrow... :)

**Comment by kevmoo** [comment]
@jonasfj 😁

**Pull request merged**